--- gavl-0.2.4/include/Makefile.am.orig	2006-06-05 21:05:24.000000000 +0200
+++ gavl-0.2.4/include/Makefile.am	2006-08-20 22:04:40.582582324 +0200
@@ -1,6 +1,6 @@
 SUBDIRS = gavl
 
-includedir = $(prefix)/lib/gavl/include
+includedir = $(libdir)/gavl/include
 
 private_headers = \
 accel.h \
--- gavl-0.2.4/configure.ac.orig	2006-07-02 13:34:15.000000000 +0200
+++ gavl-0.2.4/configure.ac	2006-08-20 22:05:09.040626042 +0200
@@ -64,7 +64,7 @@
 dnl Build optimization flags
 dnl
 
-LQT_OPT_CFLAGS($host_cpu, ["-O3 -funroll-all-loops -fomit-frame-pointer -ffast-math"])
+LQT_OPT_CFLAGS($host_cpu, ["-fomit-frame-pointer -ffast-math"])
 
 dnl
 dnl Check for SIMD
