diff -Nur gavl-0.2.4.orig/gavl/memcpy.c gavl-0.2.4.non-x86/gavl/memcpy.c
--- gavl-0.2.4.orig/gavl/memcpy.c	2006-06-26 20:32:09.000000000 +0000
+++ gavl-0.2.4.non-x86/gavl/memcpy.c	2007-05-19 17:56:01.000000000 +0000
@@ -428,11 +428,14 @@
 {
   unsigned long long int x;
 
+# if defined(ARCH_x86)
   /* that should prevent us from trying cpuid with old cpus */
   if( config_flags & GAVL_ACCEL_MMX ) {
     __asm__ volatile (".byte 0x0f, 0x31" : "=A" (x));
     return x;
-  } else {
+  } else
+#endif
+  {
     return times(NULL);
   }
 }
